core/src/main/java/com/g16/feyrune/model/Model.java

```diff
@@ -0,0 +1,31 @@
+package com.g16.feyrune.model;
+
+import com.g16.feyrune.enums.ModelState;
+import com.g16.feyrune.interfaces.IObserver;
+
+import java.awt.*;
+import java.util.ArrayList;
+
+public class Model {
+    private ModelState currentModelState;
+    private ArrayList<IObserver> observers;
+
+    public Model() {
```

Comment:
Borde inte model ha en constructor där den tar in en spelare? Annars behöver vi en funktion för getPlayer() så att CombatModel får access till spelarens data. Eller hanterar vi det annorlunda nu?

Reply:
Me dum dum